packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1982,8 +1985,8 @@ class _InactiveElements {
     assert(!element._active);
   }
 
-  bool debugContains(Element element) {
-    bool result;
+  bool? debugContains(Element element) {
+    bool? result;
```

Comment:
this can be late and the method can be non-nullable; this can throw in release builds, it's debug-only

Reply:
Done